core/src/main/java/hudson/TcpSlaveAgentListener.java

```diff
@@ -156,7 +156,7 @@ public String getAgentProtocolNames() {
 
     /**
      * Gets Remoting minimum supported version to prevent unsupported agents from connecting
-     * @since 2.169
+     * @since TODO
```

Comment:
```suggestion
     * @since 2.171
```

It's merged to master, so we know it'll be in the next weekly release.

Reply:
Thanks, I've changed according to your suggestion